arkouda/pdarrayIO.py

```diff
@@ -581,3 +588,146 @@ def save_all(columns : Union[Mapping[str,pdarray],List[pdarray]], prefix_path :
             first_iter = False
         else:
             arr.save(prefix_path=prefix_path, dataset=name, file_format=file_format, mode='append')
+
+
+def import_data(read_path: str, write_file: str = None,
+                return_obj: bool = False, index: bool = False):
+    """
+    Import data from Pandas to Arkouda readable format
+
+    Parameters
+    __________
+    read_path: str
+        path to file where pandas data is stored. This can be glob expression for parquet formats.
+    write_to: str, optional
+        path to file to write arkouda formatted data to. Only write file if provided
+    return_obj: bool, optional
+        Default False. When True return the Arkouda DataFrame object
+    index: bool, optional
+        Default False. When True, maintain the indexes loaded from the pandas file
+
+    Raises
+    ______
+    RuntimeWarning
+        - Export attempted on Parquet file. Arkouda formatted Parquet files are readable by pandas.
+    RuntimeError
+        - Unsupported file type
+
+    Returns
+    _______
+    pd.DataFrame
+        When `return_obj=True`
+
+    See Also
+    ________
+    pandas.DataFrame.to_parquet, pandas.DataFrame.to_hdf,
+    pandas.DataFrame.read_parquet, pandas.DataFrame.read_hdf,
+    ak.export
+
+    Notes
+    _____
+    - Import can only be performed from hdf5 or parquet files.
+    - Import is only designed to import data from Pandas. Additional format support will require attribute validation
```

Comment:
These two import comments seem contradictory. I believe a longer explanation is needed, or a reference to a deep-dive into the difference in how pandas and arkouda write parquet/hdf5 files.

Reply:
i don't think these are contradictory but a rewording could be nice. Maybe something like
```
- Currently import can only be performed on hdf5 and parquet files.
- Currently import only supports data written using pandas into one of the above file formats.
```

Reply:
@pierce314159 - I reworded slightly different (one bullet), but I think it captures what you are stating.